packages/firestore/test/unit/specs/spec_test_runner.ts

```diff
@@ -420,6 +429,7 @@ abstract class TestRunner {
 
   run(steps: SpecStep[]): Promise<void> {
     console.log('Running spec: ' + this.name);
+    this.connection.reset();
```

Comment:
Why put this in start() along with all the rest of the setup logic?

Reply:
Moved to start().